File: camera/converter.js
const { spawn } = require("child_process");

module.exports = (input, output) => {
  return new Promise((resolve, reject) => {
    const ff = spawn("ffmpeg", [
      "-y",
      "-fflags", "+genpts",
      "-i", input,
      "-c", "copy",
      "-movflags", "+faststart",
      output
    ]);

    ff.on("close", code => {
      if (code === 0) resolve();
      else reject(new Error("ffmpeg failed"));
    });
  });
};

File: converter.py
import subprocess
import os

def convert_to_mp4(input_file):
    output = input_file.replace(".h264", ".mp4")

    print("Converting →", output)

    subprocess.run([
        "ffmpeg",
        "-y",
        "-fflags", "+genpts",
        "-r", "60",
        "-i", input_file,
        "-c:v", "copy",
        "-movflags", "+faststart",
        output
    ])

    os.remove(input_file)
    return output
